lib/rules/no-aria-unsupported-elements.js

```diff
@@ -0,0 +1,40 @@
+import AstNodeInfo from '../helpers/ast-node-info.js';
+import Rule from './_base.js';
+
+export function ERROR_MESSAGE_ARIA_UNSUPPORTED_PROPERTY(tag, name) {
+  return `The <${tag}> element does not support the use of ARIA roles, states, and properties such as ${name}`;
+}
+const RESERVED_ELEMENTS = new Set(['html', 'meta', 'script', 'style']);
+
+export default class NoAriaUnsupportedElements extends Rule {
+  isReserved(tag) {
```

Comment:
I would omit this function and just use `RESERVED_ELEMENTS.has(tag)` directly because:
* it's trivial
* if we really wanted to keep it, we should define it outside the class since it doesn't use any class state

Reply:
Agreed. That makes it much cleaner. Thanks!